src/char/int_mercenary.cpp

```diff
@@ -90,6 +108,25 @@ bool mapif_mercenary_save(struct s_mercenary* merc)
 		flag = false;
 	}
 
+	// Save skill cooldowns
+	SqlStmt *stmt = SqlStmt_Malloc(sql_handle);
+
+	if (SQL_ERROR == SqlStmt_Prepare(stmt, "REPLACE INTO `%s` (`mer_id`, `skill`, `tick`) VALUES (%d, ?, ?)", schema_config.skillcooldown_mercenary_db, merc->mercenary_id))
+		SqlStmt_ShowDebug(stmt);
```

Comment:
If it fails to prepare, it does not really make sense to continue?
But I see that this is already wrong in the code where you copied it from.